### DISCUSSION TITLE
Latest WinUAE sync point?

### DISCUSSION THREAD

**Discussion by jacereda** [BlitterStudio/amiberry#1485] [Q&A] [answered]
Hi,

I'd like to attempt pulling some changes from WinUAE. Can you share which WinUAE commit is the last sync point for your master branch?

Thanks.

**Comment by midwan** [comment, marked as answer]
At the time of writing this, the latest WinUAE commit merged would be `0c0a444f02381647a751c4e30eae29fd9456cf3a`

**Comment by jacereda** [comment]
Great, thanks.

**Comment by solskogen** [comment]
Please use Discussions for these types of questions, not issues.
